src/storage/exec/IndexDedupNode.cpp

```diff
@@ -0,0 +1,83 @@
+/* Copyright (c) 2021 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+#include "storage/exec/IndexDedupNode.h"
+namespace nebula {
+namespace storage {
+IndexDedupNode::IndexDedupNode(const IndexDedupNode& node)
```

Comment:
I think the de-dup node here should be a public one, not used for indexes.

Reply:
Yes. But because the data iteration format of IndexNode is Row instead of KV, the processing logic is different from GetNeighbors. There are indeed plans to modify the GetNeighbors node to return Row, and then rename IndexXxxNode to XxxNode, not only Dedup, but also Limit, Sort, Aggregate, etc.

Reply:
Got， good job.